### ISSUE TITLE
Inference accuracy not reaching the original paper's results

### ISSUE THREAD

**Issue by NiuPDJ** [StijnGroenen/GazeSCRNN#1]
Thank you very much for your work. I tried to reproduce this work using your code, but the accuracy is far from the results in the paper (2.9mm, 8.9°). The parameters selected are:

GazeSCRNN-Events1750-FPTT-Backprop8-TWT20
--num_users 48
--data_preload
--gpus 0
-b 64
--fptt
--backprop_timesteps 8
--data_event_count 1750
--data_augment_affine
--data_augment_noise 0.1
--time_window_threshold 20000.

Could you please tell me where the settings are wrong? Or are there any other tricks I can refer to during training?
Looking forward to your reply, thank you very much!

**Comment by StijnGroenen** [comment]
Hi!

Thank you for your interest in reproducing the results and for providing the detailed configuration.

You should turn off data augmentation (by removing the --data_augment_affine and --data_augment_noise arguments). I experimented with these options, but found that they actually reduce model accuracy, so they aren’t used in the final results reported in the paper.

Once you disable these, you should get results much closer to the reported metrics.

**Comment by NiuPDJ** [comment]
> Hi!
> 
> Thank you for your interest in reproducing the results and for providing the detailed configuration.
> 
> You should turn off data augmentation (by removing the --data_augment_affine and --data_augment_noise arguments). I experimented with these options, but found that they actually reduce model accuracy, so they aren’t used in the final results reported in the paper.
> 
> Once you disable these, you should get results much closer to the reported metrics.

Thank you very much for your reply. Your work has inspired me a lot.
